src/Stack/Docker.hs

```diff
@@ -78,7 +78,7 @@ getCmdArgs
 getCmdArgs docker imageInfo isRemoteDocker = do
     config <- view configL
     deUser <-
-        if fromMaybe (not isRemoteDocker) (dockerSetUser docker)
+        if fromMaybe (not isRemoteDocker) (dockerSetUser docker) && not osIsWindows
```

Comment:
I will check if this is the right thing to do and if yes then provide justification.

Reply:
It is the right thing to do - dockerSetUser feature won't work on Windows.